### ISSUE TITLE
Rewrite _analyze examples to use JSON body

### ISSUE THREAD

**Issue by damienalexandre** [jolicode/elasticsearch-cheatsheet#11]
```
GET /blog/_analyze?field=title&text=powerful
```
Is deprecated. We should display:
```
GET /_analyze
{
  "text": "向日葵",
  "tokenizer": "icu_tokenizer"
}
```

**Comment by shangyilim** [comment]
do you need help updating the html example for the issue mentioned above?

**Comment by damienalexandre** [comment]
Yes! All the help is welcome 🙏 We accept pull requests 😉

**Comment by damienalexandre** [comment]
Fixed in #15
